### PR TITLE
upgrade core/proto to fix broken proto package naming

### PR DESCRIPTION
**Pull request by jeffgrunewald**
Includes the latest helium_proto definitions and fixes for grpcbox server behaviour modules in blockchain-core's `transaction` service and node's own `follower` service required for to address build breakage as a result of the changes to the proto3 package names for both service definitions (grpcbox_plugin generates the behaviour module from a template constructing the module name by joining the package name components with the file name).

This change also includes a stub definition of a forthcoming `active_gateways/2` grpc stream API attached to the follower service to avoid any `warnings as errors` CI build errors for the API as defined in the latest proto file but which still requires implementation on the core and node side.

### CODE REVIEW

File: src/helium_follower_service.erl [hunk 1]
```diff
@@ -5,12 +5,17 @@
 %%%-------------------------------------------------------------------
 -module(helium_follower_service).
 
--behaviour(helium_follower_bhvr).
+-behaviour(helium_follower_follower_bhvr).
```
Comment (jadeallenx) [inline src/helium_follower_service.erl:8]: this isn't a typo?

Comment (jeffgrunewald) [reply src/helium_follower_service.erl:8]: unfortunately not; the proto file is `follower.proto` and the package defined in there is `helium.follower` so when grpcbox_plugin constructs this behaviour it's calling it that by joining the package name and the file name and sticking `bhvr` on the end

Comment (jeffgrunewald) [reply src/helium_follower_service.erl:8]: i didn't see any way to override the naming convention in the config without modifying our fork of grpcbox_plugin and i didn't think these are expected to live long enough to be worth that

Comment (jadeallenx) [reply src/helium_follower_service.erl:8]: OK lol

**Review by jadeallenx — APPROVED** [review]

**Pull request merged**